src/main.test.ts

```diff
@@ -132,6 +132,75 @@ test('it sets the updated dependency as an output for subsequent actions', async
   expect(core.setOutput).toBeCalledWith('cvss', 0)
 })
 
+test('it sets the updated dependency as an output for subsequent actions when there is a leading v in the commit message version', async () => {
```

Comment:
This is exactly why we should preserve the leading `"v"`... the user may do post-processing and I'd rather leave the decision to "strip" any prefix up to them...